assets/js/components/TourTooltips.js

```diff
@@ -149,6 +149,7 @@ export default function TourTooltips( { steps, tourID, gaEventCategory } ) {
 	 * @property {string} type   Specific type (tour, step, beacon).
 	 *
 	 * @since 1.28.0
+	 * @since n.e.x.t Calls new callback prop.
```

Comment:
@eugene-manuilov I am not sure what the suggestion is here, I can't spot the difference 😅

Reply:
@johnPhillips an additional line between @since and @see tags

Reply:
Thanks. I get the following linting errors though.
```
[eslint sitekit/jsdoc-tag-order] The @since tag should not have a newline between it and the following @see tag.
[eslint sitekit/jsdoc-newlines] There should not be an empty line between @since and @see.
```

Reply:
Oh... I didn't see it before.. weird... :thinking:

Reply:
Ok, please, disregard this comment then :smile: